src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetDateCommand.cs

```diff
@@ -40,14 +41,38 @@ public DateTime Date
             }
         }
 
+        private DateTime _date;
+        private bool _dateSpecified;
+
+        // DateTimeOffset.MinValue.ToUnixTimeSeconds()
```

Comment:
Looks like commented code. Please reword like:
```suggestion
        // The const comes from DateTimeOffset.MinValue.ToUnixTimeSeconds()
```